src/main/scala/tech/cryptonomic/conseil/metadata/MetadataService.scala

```diff
@@ -17,6 +18,14 @@ class MetadataService(config: PlatformsConfiguration,
                       cacheConfiguration: AttributeValuesCacheConfiguration,
                       tezosPlatformDiscoveryOperations: TezosPlatformDiscoveryOperations) {
 
+  // inits attributes cache
+  def initAttributesCache(): Unit =
```

Comment:
Does it support deployment on many instances right now? In order to initialize the cache, we'll need to hit each instance which could be problematic.

Reply:
why not? you just use instances IP to init cache

Reply:
Yeah, but it doesn't scale well since you need to be aware of IP addresses of all machines. The idea of load balancing is to treat the whole application as if it was a single instance.

Reply:
I agree, but this operation either way should be done once. That was a requirement for those changes.